src/store/actions/actionTypes.js

```diff
@@ -0,0 +1,4 @@
+export const TOGGLE_CHAT = 'TOGGLE_CHAT';
+export const ADD_NEW_USER_MESSAGE = 'ADD_NEW_USER_MESSAGE';
+export const ADD_NEW_RESPONSE_MESSAGE = 'ADD_NEW_RESPONSE_MESSAGE';
+export const ADD_NEW_LINK_SNIPPET = 'ADD_NEW_LINK_SNIPPET';
```

Comment:
We usually separate actions by reducer. We also "scope" them like `export const TOGGLE_CHAT = 'BEHAVIOUR_REDUCER/TOGGLE_CHAT'`.

Let me know what you think, maybe it's overkill here.

Reply:
I don't think it's a bad decision, but since it's just a string, what difference does it make? I mean, you give the sense of "scope" to the action, but the reducers are doing that, and it's just a string telling you to which reducer it belongs :thinking:

Reply:
Yeah is mostly for debugging.